.github/workflows/ci.yml

```diff
@@ -57,12 +31,75 @@ jobs:
           --health-timeout 5s
           --health-retries 10
     steps:
-    - uses: actions/checkout@v3.5.0
-    - name: SBT
-      uses: ./.github/actions/sbt
-      env:
-        USE_DOCKER_FOR_TESTS: false
-        ES6_TEST_URL: http://elasticsearch:9200
-        LOCALSTACK_ENDPOINT: http://localstack:4566
-      with:
-        args: clean compile test:compile test
+      - uses: actions/checkout@v3
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+      - name: Kahuna (client-side)
+        working-directory: ./kahuna
+        run: |
+          npm install-clean
+          npm run undist
+          npm test
+          npm run dist
+      - name: SBT
+        uses: ./.github/actions/sbt
+        env:
+          USE_DOCKER_FOR_TESTS: false
+          ES6_TEST_URL: http://elasticsearch:9200
+          LOCALSTACK_ENDPOINT: http://localstack:4566
+        with:
+          args: clean compile test:compile test debian:packageBin
+      - uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
There's a later version of this action, [v4](https://github.com/aws-actions/configure-aws-credentials/releases), which I'd hope would be compatible with Node 16.

Reply:
done, thanks for the heads up